src/manager/manager.go

```diff
@@ -139,12 +148,13 @@ func (z *ZarManager) WalkDir(dir string, foldername string, root bool) {
 
 // TODO: Change to interface for Metadata to have diff types of Metadata
 // IncludeFolderBegin implements Manager.IncludeFolderBegin
-func (z *ZarManager) IncludeFolderBegin(name string) {
+func (z *ZarManager) IncludeFolderBegin(name string, mod_time int64) {
         h := &FileMetadata{
                         Begin   : -1,
                         End     : -1,
                         Name    : name,
                         Type    : Directory,
+		                ModTime	: mod_time,
```

Comment:
Something strange here:
I have thought it is the problem of indentation but it seems not.
I try to delete all leading spaces/tabs in vim but git diff still shows:
         h := &FileMetadata{
-                        Begin   : -1,
-                        End     : -1,
-                        Name    : name,
-                        Type    : Directory,
-                               ModTime : mod_time,
+            Begin   : -1,
+            End     : -1,
+            Name    : name,
+            Type    : Directory,
+                   ModTime     : mod_time,
         }

It seems some strange characters after "Directory,". I deleted the line break and enter again. It looks normal now.

@@ -150,11 +150,11 @@ func (z *ZarManager) WalkDir(dir string, foldername string, mod_time int64, root
 // IncludeFolderBegin implements Manager.IncludeFolderBegin
 func (z *ZarManager) IncludeFolderBegin(name string, mod_time int64) {
         h := &FileMetadata{
-                        Begin   : -1,
-                        End     : -1,
-                        Name    : name,
-                        Type    : Directory,
-                               ModTime : mod_time,
+            Begin   : -1,
+            End     : -1,
+            Name    : name,
+            Type    : Directory,
+            ModTime    : mod_time,
         }

I will fix now but just let you know this problem.